packages/css/src/link/link.scss

```diff
@@ -5,14 +5,22 @@
 
 @import "../../utils/calculate-fluid-style";
 
+@mixin reset {
+  background-color: transparent;
```

Comment:
Which style does this override?

Reply:
[It's this part of normalize.css](https://github.com/necolas/normalize.css/blob/master/normalize.css#L72-L78)

Reply:
I don’t think we’ll be supporting IE 10, let’s remove this.